fitbenchmarking/fitbenchmarking
# Move start_matlab call so it is only called once

#### Description of Work

Fixes an issue where memory was being accumulated by matlab controllers


#### Testing Instructions

1. Run some Matlab tests on large datasets. Check that memory is not reserved by matlab.

Function: Does the change do what it's supposed to?
Tests: Does it pass? Is there adequate coverage for new code?
Style: Is the coding style consistent? Is anything overly confusing?
Documentation: Is there a suitable change to documentation for this change?
